packages/teams-js/src/public/media.ts

```diff
@@ -232,23 +254,26 @@ export namespace media {
       function handleGetMediaCallbackRequest(mediaResult: MediaResult): void {
         if (callback) {
           if (mediaResult && mediaResult.error) {
-            /* eslint-disable-next-line strict-null-checks/all */ /* Fix tracked by 5730662 */
-            callback(mediaResult.error, null);
+            callback(mediaResult.error, new Blob());
           } else {
             if (mediaResult && mediaResult.mediaChunk) {
               // If the chunksequence number is less than equal to 0 implies EOF
               // create file/blob when all chunks have arrived and we get 0/-1 as chunksequence number
               if (mediaResult.mediaChunk.chunkSequence <= 0) {
                 const file = createFile(helper.assembleAttachment, helper.mediaMimeType);
-                callback(mediaResult.error, file);
+                callback(mediaResult.error, file ?? new Blob());
               } else {
                 // Keep pushing chunks into assemble attachment
-                const assemble: AssembleAttachment = decodeAttachment(mediaResult.mediaChunk, helper.mediaMimeType);
-                helper.assembleAttachment.push(assemble);
+                const assemble: AssembleAttachment | null = decodeAttachment(
+                  mediaResult.mediaChunk,
+                  helper.mediaMimeType,
+                );
+                if (assemble) {
+                  helper.assembleAttachment.push(assemble);
+                }
```

Comment:
since this if statement is new, does it warrant some sort of error or log statement if `assemble` is null? Seems unlikely we had cases like that that we weren't noticing, but not impossible

Reply:
Good suggestion; I have added a logging statement to indicate when this situation has arisen.